packages/infrastructure/aws_eks/vars.tf

```diff
@@ -109,6 +109,11 @@ variable "node_security_groups" {
   default     = []
 }
 
+variable "node_ebs_volume_size" {
```

Comment:
Generally, we suffix the quantity onto these types of inputs. So `node_ebs_volume_size_gb`